util/buildinfo/export.go

```diff
@@ -0,0 +1,43 @@
+package buildinfo
+
+import (
+	"github.com/pkg/errors"
+)
+
+// ExportMode represents the export mode for buildinfo opt.
+type ExportMode byte
+
+const (
+	// ExportNone doesn't export build dependencies.
+	ExportNone ExportMode = 1 << iota
```

Comment:
My thinking in here was
```
ExportNone = 0
ExportImageConfig = 1
ExportMetadata = 1 << 1
ExportAll = ExportImageConfig | ExportMetadata
```

That should simplify the checks. `ExportUnknown` is already unused.

Reply:
Done

Reply:
`ExportNone` should be 0